docs.html

```diff
@@ -30,48 +30,84 @@
   <body>
     <div class="doc-page-layout">
       <!-- Navbar -->
-      <header class="navbar">
-        <nav class="container-xxl">
-          <a class="nav-brand" href="#">
-            <span class="nav-brand__maintxt">K</span>
-            <span class="nav-brand__sectxt">ube UI</span>
-          </a>
-          <button class="navbar-toggler" id="btn-toggle">
-            <span class="material-icons"> menu </span>
-          </button>
-          <div class="navbar-collapse collapse" id="nav-toggle">
-            <ul class="navbar-nav">
+      <header class="navbar navbar-home w-full"> 
+        <nav class="navbar-wrapper">
+          <div class="navbar-left-aligned">
+            <!-- <svg
+              class="sidebar-toggler"
+              id="sidebar-toggler"
+              viewBox="0 0 100 40"
+              width="40"
+              height="40"
+            >
+              <rect width="60" height="10"></rect>
+              <rect y="20" width="60" height="10"></rect>
+              <rect y="40" width="60" height="10"></rect>
+            </svg> -->
+            <div class="sidebar-toggler"></div>
+            <a class="nav-brand" href="#">
+              <span class="nav-brand__maintxt primary-text">K</span>
+              <span class="nav-brand__sectxt">ube UI</span>
+            </a>
+           
+            <ul class="navbar-nav navbar-nav-collapse">
               <li class="nav-item">
-                <a class="nav-item-link p-4 bold" href="#">Home</a>
+                <a class="nav-item-link" href="./index.html">Home</a>
               </li>
               <li class="nav-item">
-                <a class="nav-item-link p-4 bold" href="#">Documentation</a>
+                <a
+                  class="nav-item-link"
+                  href="#"
+                  >Documentation</a
+                >
               </li>
             </ul>
-            <ul class="navbar-nav align-end">
+          </div>
+          <div class="navbar-right-aligned">
+            <ul class="navbar-nav navbar-fixed">
               <li class="nav-item">
-                <span class="nav-item-link">
-                  <i class="fab fa-github p-4" aria-hidden="true"></i>
-                </span>
+                <a class="nav-item-link" href="#">
+                  <i class="fab fa-linkedin-in" aria-hidden="true"></i>
+                </a>
               </li>
               <li class="nav-item">
-                <span class="nav-item-link">
-                  <i class="fab fa-twitter p-4" aria-hidden="true"></i>
-                </span>
+                <a
+                  class="nav-icon-link badge-container no-link-style"
+                  href="https://github.com/SHIKHASINGH1506"
```

Comment:
add `target="_blank" or else the URL will open in the same tab the user is on, which is not a good UX in this case